### ISSUE TITLE
Unable to find 'getBlock' in 'net.minecraft.world.level.block.state.IBlockData'

### ISSUE THREAD

**Issue by x1p** [SBPrime/AsyncWorldEdit#417]
I have a lot of spam at startup on 1.18.
```
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] Enabling AsyncWorldEdit v3.9.3
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] Platform set to Bukkit API
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] ===================================================================================
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] =     ___                         _       __           __    ________    ___ __   =
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] =    /   |  _______  ______  ____| |     / /___  _____/ /___/ / ____/___/ (_) /_  =
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] =   / /| | / ___/ / / / __ \/ ___/ | /| / / __ \/ ___/ / __  / __/ / __  / / __/  =
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] =  / ___ |(__  ) /_/ / / / / /__ | |/ |/ / /_/ / /  / / /_/ / /___/ /_/ / / /_    =
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] = /_/  |_/____/\__, /_/ /_/\___/ |__/|__/\____/_/  /_/\__,_/_____/\__,_/_/\__/    =
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] =             /____/                                                              =
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] = Author: SBPrime                                                                 =
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] =                                                                                 =
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] ===================================================================================
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] Unable to find 'getBlock' in 'net.minecraft.world.level.block.state.IBlockData'
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] Unable to find 'getBlock' in 'net.minecraft.world.level.block.state.IBlockData'
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] Unable to find 'getBlock' in 'net.minecraft.world.level.block.state.IBlockData'
[21:11:05] [Server thread/INFO]: [AsyncWorldEdit] Unable to find 'getBlock' in 'net.minecraft.world.level.block.state.IBlockData'

...This line gets repeated 898 times in total, then it continues...

[21:11:06] [Server thread/INFO]: [AsyncWorldEdit] Unable to find 'getBlock' in 'net.minecraft.world.level.block.state.IBlockData'
[21:11:06] [Server thread/INFO]: [AsyncWorldEdit] Initializing WorldEdit.
[21:11:07] [AWE:Cron:undoCleanup/INFO]: [AsyncWorldEdit] Undo cleanup started...
[21:11:07] [AWE:Cron:undoCleanup/INFO]: [AsyncWorldEdit] ...undo cleanup done.
[21:11:07] [Server thread/INFO]: [AsyncWorldEdit] No matching adapter found for Bukkit.v1_18_R1, DirectChunkAPI disabled
[21:11:07] [Server thread/INFO]: [AsyncWorldEdit] PlotMeFix set to class org.primesoft.asyncworldedit.plotme.NullFix.
[21:11:07] [Server thread/INFO]: [AsyncWorldEdit] Enabled
```
AWE v3.9.3
WE v7.2.8-SNAPSHOT+6003-6e106c1
Paper 1.18 build 47

**Comment by trustiestbullet** [comment]
[18:32:31] [Server thread/INFO]: [AsyncWorldEdit] Unable to find 'getBlock' in 'net.minecraft.world.level.block.state.IBlockData'
same on spigot 1.18

**Comment by Dymeth** [comment]
Same

**Comment by ErythroCraft** [comment]
https://github.com/SBPrime/AsyncWorldEdit/issues/419

**Comment by QuantumZizo** [comment]
Same

**Comment by PapaSchlumpf71** [comment]
how can fix it?
[Server thread/INFO]: [AsyncWorldEdit] Unable to find 'getBlock' in 'net.minecraft.world.level.block.state.IBlockData'

**Comment by waddon1** [comment]
Same

**Comment by Bloodlex** [comment]
same for me

**Comment by Skidamek** [comment]
same

**Comment by Jeremy026** [comment]
Same

**Comment by Michel3951** [comment]
I solved this by just removing the plugin 👍

**Comment by ErythroCraft** [comment]
Here are all updates and infos why AsynchWorldEdit is not working on 1.18.1 servers.
**An update for AsynchWorldEdit to 1.18.1 is required, because MC1.18 has not the Log4J-2 update.**

Lates stable version of WorldEdit 7.2.x on GitHub is https://github.com/EngineHub/WorldEdit/tree/version/7.2.x

Lates stable version of WorldEdit 7.2.x on DevBukkit is https://dev.bukkit.org/projects/worldedit/files/3559523

Lates stable version of WorldEdit 7.2.9 on EngineHub is https://builds.enginehub.org/job/worldedit/19521
Lates stable version of WorldEdit 7.3.0 on EngineHub is https://builds.enginehub.org/job/worldedit/19235

Default is Java17 because the Log4J was updated to Log4J-2
Java17: https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html
Log4J-2: https://logging.apache.org/log4j/2.x/

**Comment by Michel3951** [comment]
> 

this is everything except related to the issue

**Comment by Michel3951** [comment]
> Here are all updates and infos why AsynchWorldEdit is not working on 1.18.1 servers. **An update for AsynchWorldEdit to 1.18.1 is required, because MC1.18 has not the Log4J-2 update.**
> 
> Lates stable version of WorldEdit 7.2.x on GitHub is https://github.com/EngineHub/WorldEdit/tree/version/7.2.x
> 
> Lates stable version of WorldEdit 7.2.x on DevBukkit is https://dev.bukkit.org/projects/worldedit/files/3559523
> 
> Lates stable version of WorldEdit 7.2.9 on EngineHub is https://builds.enginehub.org/job/worldedit/19521 Lates stable version of WorldEdit 7.3.0 on EngineHub is https://builds.enginehub.org/job/worldedit/19235
> 
> Default is Java17 because the Log4J was updated to Log4J-2 Java17: https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html Log4J-2: https://logging.apache.org/log4j/2.x/

this is not related to the issue

**Comment by SBPrime** [comment]
As stated on spigot this is a know issue, and it causes no problem, except the log. That said a fix was applied in 3.9.4.

**Comment by ErythroCraft** [comment]
Is ok.

I'm using FAWE since yesterday. 
It's working with all my installed plugins, paper and mods on my server.
The payments of my private accounts will be deleted next month.

Have fun and a long life.
